### DISCUSSION TITLE
页面读取深度depth设置在前端展示

### DISCUSSION THREAD

**Discussion by Eddie-Hyu** [Qexo/Qexo#584] [Ideas]
由于页面做了分类，部分页面层级>=2，在代码中修改depth无效且不太方便，是否有机会做到Qexo的设置中，自行跳转数据查询深度。

**Comment by am-abudu** [comment]
这个比较复杂，并且难以向一般用户解释。请自行修改代码，并在保存后重启程序即可生效
